cms/djangoapps/contentstore/tests/test_proctoring.py

```diff
@@ -85,7 +89,8 @@ def test_publishing_exam(self, is_time_limited, default_time_limit_minutes,
             is_proctored_exam=is_proctored_exam,
             is_practice_exam=is_practice_exam,
             due=datetime.now(UTC) + timedelta(minutes=default_time_limit_minutes + 1),
-            exam_review_rules="allow_use_of_paper"
+            exam_review_rules="allow_use_of_paper",
+            hide_after_due=False,
```

Comment:
In this test case, add a parameter for hide_after_due and make it true in the timed exam case (so we have some coverage for it being true).